indigo_app/views/places.py

```diff
@@ -833,13 +833,15 @@ def facet_subtype(self, facets, qs):
                 st.name,
                 st.abbreviation,
                 counts.get(st.abbreviation, 0),
-                self.form.cleaned_data.get("subtype") == st.abbreviation
+                st.abbreviation in self.form.cleaned_data.get("subtype")
```

Comment:
```suggestion
                st.abbreviation in self.form.cleaned_data.get("subtype", [])
```

to avoid errors in case of no value